roc/version.go

```diff
@@ -87,6 +87,9 @@ func checkVersion() {
 	if atomic.CompareAndSwapInt32(&versionCheckOnce, 0, 1) {
 		vi := fetchVersion()
 
+		logWrite(LogDebug, "entering versionCheck()")
+		defer logWrite(LogDebug, "leaving versionCheck(): version=%+v", vi)
```

Comment:
As I already wrote before, there is no need for entering/leaving message pair here, we can just log version after fetching it and before validating it. There is no big work done between enter and leave, unlike other methods.

Also using defer hurts here, because this way Validate() will panic before version is logged.